feed.xml

```diff
@@ -37,7 +37,36 @@
 				<link>{{ site.url }}{{ page.url }}</link>
 				<guid isPermaLink="true">{{ site.url }}{{ page.url }}</guid>
 			</item>
-          {% endif %}
         {% endfor %}
+
+				{% assign pages = site.pages | where:'layout','lesson'| where: 'lang', 'en' | sort:'date' | reverse %}
+				{% for page in pages limit:10 %}
+			<item>
+				<title>{{ page.title }}</title>
+				{% if page.abstract %}
+					<description>{{ page.abstract | xml_escape }}</description>
+				{% else %}
+								<description>{{ page.excerpt | xml_escape }}</description>
+				{% endif %}
+				<pubDate>{{ page.date | date: "%a, %d %b %Y %H:%M:%S %z" }}</pubDate>
+				<link>{{ site.url }}{{ page.url }}</link>
+				<guid isPermaLink="true">{{ site.url }}{{ page.url }}</guid>
+			</item>
+				{% endfor %}
+
+				{% assign pages = site.pages | where:'layout','lesson'|  where:'lang','es' | sort:'translation_date' | reverse %}
```

Comment:
I don't think it makes sense to loop the Spanish lessons distinctly from the English lessons _if they're both ending up in the same feed_.

Either we should stream all the lessons and posts together (so finding all lessons without filtering for `lang == 'en'`) or we should create separate RSS feeds for lessons by language (and separate the blog posts, too). If the latter, we should make sure to use https://www.w3schools.com/XML/rss_tag_language.asp

I'm a fan of [KISS](https://en.wikipedia.org/wiki/KISS_principle) as we don't publish with ferocious frequency, and given how few users subscribe to our RSS, I don't think we have the demand for separating them into different streams right now.